src/wp-admin/includes/misc.php

```diff
@@ -1398,6 +1398,15 @@ function wp_admin_canonical_url() {
 	// Ensure we're using an absolute URL.
 	$current_url  = set_url_scheme( 'http://' . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'] );
 	$filtered_url = remove_query_arg( $removable_query_args, $current_url );
+
+	/**
+	 * Filters the admin canonical url value
+	 *
+	 * @since 6.3.2
```

Comment:
We don't introduce any new feature in a minor release, let's target 6.4.0 instead :)

Reply:
I'd say 6.5 as this is an enhancement still.